litestar/logging/config.py

```diff
@@ -188,23 +200,22 @@ class LoggingConfig(BaseLoggingConfig):
     Filter instance.
     """
     propagate: bool = field(default=True)
-    """If messages must propagate to handlers higher up the logger hierarchy from this logger."""
-    formatters: dict[str, dict[str, Any]] = field(
-        default_factory=lambda: {
-            "standard": {"format": "%(levelname)s - %(asctime)s - %(name)s - %(module)s - %(message)s"}
-        }
-    )
+    """If messages must propagate to handlers higher up the logger hierarchy from this logger.
+
+    .. deprecated:: 2.10.0
+        This parameter is deprecated. It will be removed in a future release. Use `propagate` at the logger level.
+    """
+    formatters: dict[str, dict[str, Any]] = field(default_factory=_get_default_formatters)
+    """A dict in which each key is a formatter and each value is a dict describing how to configure the corresponding
```

Comment:
```suggestion
    """A :term:`dict` in which each key is a formatter and each value is a dict describing how to configure the corresponding
```

Reply:
I'm reverting these ones (if you agree?) because it would link to https://litestar-org.github.io/litestar-docs-preview/3543/reference/datastructures/index.html#litestar.datastructures.Cookie.dict

Also, the link to dict is just above, in the type signature.